src/VersionHandler.go

```diff
@@ -51,15 +53,7 @@ func getOSArch() string {
 		}
 		return strings.TrimSuffix(string(out), "\n")
 	case windows:
-		out, err := exec.Command("if exist \"%ProgramFiles(x86)%\" echo 64-bit").Output()
-		if err != nil {
-			fmt.Println("error occured")
-			fmt.Printf("%s", err)
-		}
-		if string(out) == "64-bit" {
-			return "x86_64"
-		}
-		return "i686"
+		return "x86_64"
```

Comment:
Für Windows findest du die Info in 

    $ echo %PROCESSOR_ARCHITECTURE%

bzw.

    $ [environment]::Is64BitOperatingSystem